doc/man3/ASN1_STRING_length.pod

```diff
@@ -77,13 +77,33 @@ character in big endian format, UTF8String will be in UTF8 format.
 Similar care should be take to ensure the data is in the correct format
 when calling ASN1_STRING_set().
 
+=head1 RETURN VALUES
+
+ASN1_STRING_length() returns the length of the content of B<x>.
+
+ASN1_STRING_get0_data() and ASN1_STRING_data() return an internal pointer to
+the data of B<x>.
```

Comment:
Should we say something about who is responsible for freeing this data?

Reply:
This is described in the DESCRIPTION section, then still need to re-describe here?

Reply:
Ok - no need if we say it elsewhere.